baton/hub.go

```diff
@@ -40,45 +40,57 @@ func (h hub) run() {
 				close(c.send)
 			}
 		case rawMsg := <-h.receive:
-			fmt.Println(rawMsg)
 			var cmd command
 			if err := json.Unmarshal(rawMsg.data, &cmd); err != nil {
 				fmt.Println("nu")
 				log.Println(err)
```

Comment:
Perhaps we could change this to the following to be a bit more descriptive?

    log.Println("error unmarshaling incoming message:", err)